procurement_purchase_no_grouping/tests/test_procurement_purchase_no_grouping.py

```diff
@@ -79,3 +82,38 @@ def test_procurement_no_grouping_order_purchase(self):
             2,
             "Procured purchase orders lines are the same",
         )
+
+    def test_procurement_system_grouped_purchase(self):
+        self.category.procured_purchase_grouping = None
+        self._set_system_grouping("standard")
+        self._run_procurement()
+        self._run_procurement()
```

Comment:
Why 2 times?

Reply:
Honestly - I wouldn't know. That's the way it is in original code. Was wondering when debugging why but let it go. Should that be limited to one?

Reply:
Yes, try to put only one and see if tests still pass

Reply:
OK.

Reply:
So it seems the double run was required.